nameko/constants.py

```diff
@@ -3,6 +3,7 @@
 RPC_EXCHANGE_CONFIG_KEY = 'rpc_exchange'
 SERIALIZER_CONFIG_KEY = 'serializer'
 HEARTBEAT_CONFIG_KEY = 'HEARTBEAT'
+SSL_CONFIG_KEY = 'ssl'
```

Comment:
i'm less worried about the name of the constant (though your suggestion is good) than the value of the key that needs to go into `config.yml`, currently just `ssl`